web/packages/teleport/src/services/storageService/storageService.ts

```diff
@@ -102,6 +102,21 @@ export const storageService = {
     return time ? time : 0;
   },
 
+  setLoginTime(time: number) {
+    const existingTime = Number(
+      window.localStorage.getItem(KeysEnum.LOGIN_TIME)
+    );
+    // Only set the login time if it doesn't already exist.
+    if (!existingTime) {
+      window.localStorage.setItem(KeysEnum.LOGIN_TIME, `${time}`);
+    }
+  },
+
+  getLoginTime() {
+    const time = Number(window.localStorage.getItem(KeysEnum.LOGIN_TIME));
+    return time ? time : 0;
```

Comment:
I'd return a `Date` object, not a number. It will be more useful for the callsites.